docs/source/notebooks/bayesian_mmm/requirements.txt

```diff
@@ -0,0 +1,176 @@
+anyio==3.7.1
```

Comment:
remove this as the examples should work with pymc-marketing and minimal dependencies (why beautifulsoup4 for example?)